metadata-integration/java/datahub-client/src/main/resources/MetadataChangeProposal.avsc

```diff
@@ -0,0 +1,164 @@
+{
```

Comment:
This pins us to a particular model from the MetadataChangeProposal instead of using the generated one. Was there no other way to get the CI passing?

Reply:
This file is just placeholder. It will get replaced by latest avsc during jar build.